### PR TITLE
fix: editable Typography.Title should match font size while editing

### PR DESCRIPTION
**Pull request by heiyu4585**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

close https://github.com/ant-design/ant-design/issues/34088
### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     Fix Typography.Title didn't keep font size when become editable.     |
| 🇨🇳 Chinese |      修复 Typography.Title 进入编辑模式时大小不一致的问题。    |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by heiyu4585** [comment]
CI报错如下,但是未修改该页面,帮忙看下 @afc163 
```
site/theme/template/Layout/Header/SearchBar.tsx(181,12): error TS2741: Property 'appId' is missing in....
```

**Comment by heiyu4585** [comment]
> Try to rebase feature branch.

done

**Comment by afc163** [comment]
点击 edit 的时候，上下会跳动，看看是不是合理的。

File: components/typography/style/index.less [hunk 1]
```diff
@@ -77,6 +82,14 @@
     }
   }
 
+  &-h1,
+  &-h2,
+  &-h3,
+  &-h4,
+  &-h5{
```
Comment (afc163) [inline components/typography/style/index.less:89]:
```suggestion
  &-h5 {
```

**Comment by heiyu4585** [comment]
> 点击 edit 的时候，上下会跳动，看看是不是合理的。https://preview-34169-ant-design.surge.sh/components/typography/#components-typography-demo-interactive

done

File: components/typography/style/index.less [hunk 2]
```diff
@@ -43,26 +43,36 @@
   }
 
   h1&,
+  &-h1,
+  &-h1 textarea,
```
Comment (afc163) [inline components/typography/style/index.less:47]:
```suggestion
  &-h1 > textarea,
```

File: components/typography/style/index.less [hunk 3]
```diff
@@ -198,6 +230,10 @@
       right: 10px;
       bottom: 8px;
       color: @text-color-secondary;
+      // default style
+      font-weight: normal;
+      font-size: 14px;
```
Comment (afc163) [inline components/typography/style/index.less:235]:
```suggestion
      font-size: @font-size-base;
```

File: components/typography/Editable.tsx [hunk 4]
```diff
@@ -108,21 +110,24 @@ const Editable: React.FC<EditableProps> = ({
...
+    textClassName,
   );
 
   return (
     <div className={textAreaClassName} style={style}>
       <TextArea
         ref={ref as any}
         maxLength={maxLength}
-        value={current}
+        value={current.trim()}
```
[the remark this answers is not in the document]
Comment (heiyu4585) [reply components/typography/Editable.tsx:130]: 不加前面会有个空格

File: components/typography/style/index.less (hunk 2 above)
Comment (heiyu4585) [reply components/typography/style/index.less]: 样式我再整理整理.

Comment (heiyu4585) [reply components/typography/style/index.less]: done

**Review by afc163 — APPROVED** [review]

**Pull request merged**